h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -594,8 +594,7 @@ \subsubsection{Negative Binomial Models}
 The  negative log likelihood $L(y_i,\mu_i)$ function is:
 
 \begin{multline*}
-$$^\text{max}_{\beta,\beta_0} \bigg[ \frac{-1}{N} \sum_{i=1}^{N}  \bigg \{ \bigg( \sum_{j=0}^{y_i-1} \text{log}(j + \theta^{-1} ) \bigg) - \text{log} (\Gamma (y_i + 1)) - (y_i + \theta^{-1}) \text{log} (1 + \alpha\mu_i) \\
-+ y_i \text{log}(\mu_i) + y_i \text{log} (\theta) \bigg \} \bigg]$$
+$$L(y_i,\mu_i) = \frac{-1}{N} \sum^{N-1}_{i=0} \big(\log \big(\Gamma (y_i + \theta^{-1}) \big) - \log \big(\Gamma (\theta^{-1}) \big) - \log \big(\Gamma (y_i + 1) \big) + y_i \log(\theta \mu_i) - (y_i + \theta - 1) \log (1 + \theta \mu_i) \big)$$
 \end{multline*}
 
```

Comment:
hannah:  This is totally my fault, the correct formula is:

![image](https://user-images.githubusercontent.com/7231712/208186592-22f7abe2-0484-4c82-be46-0cfbfe4b996d.png)